src/main/java/org/quantumbadger/redreader/http/HTTPBackend.java

```diff
@@ -92,16 +92,10 @@ public static String encodeList(final List<PostField> fields) {
 	}
 
 	interface Listener {
+		void onError(@CacheRequest.RequestFailureType int failureType, Throwable exception, Integer httpStatus);
 
-		void onError(RequestFailureType failureType, Throwable exception, Integer httpStatus);
-
-		void onSuccess(
-				String mimetype,
-				Long bodyBytes,
-				InputStream body);
+		void onSuccess(String mimetype, Long bodyBytes, InputStream body);
```

Comment:
All the unnecessary formatting changes are making this very hard to review :( They would also make it very hard for other people to merge in changes they've been working on in the meantime.